packages/sdk/src/evm/core/sdk.ts

```diff
@@ -9,7 +9,7 @@ import {
   PREBUILT_CONTRACTS_MAP,
   getContractTypeForRemoteName,
 } from "../contracts";
-import { SmartContract } from "../contracts/smart-contract";
```

Comment:
dont think you need to do an alias here for the type. since the real import below is dynamic it should not clash

Reply:
got it, will update

Reply:
@joaquim-verges Lint issue if they have the same name :(
![image](https://github.com/thirdweb-dev/js/assets/26052673/88804383-a80c-4abf-92d0-3639984842f4)

Reply:
Hm ok my bad then!